runtime/doc/todo.txt

```diff
@@ -2784,7 +2784,7 @@ Most interesting new features to be added when all bugs have been fixed:
     Especially when using the scrollbar.  Typing a cursor-movement command
     scrolls back to where the cursor is.
 -   Scroll commands by screen line. g CTRL-E and g CTRL-Y ?  Requires the
-    first line to be able to start halfway.
+    first line to be able to start halfway through.
```

Comment:
This one confused me. I honestly just have no idea what the original intention was so I guessed.